lib/src/main/java/com/auth0/android/lock/views/LogInFormView.java

```diff
@@ -179,7 +179,14 @@ private void resetDomain() {
     }
 
     private String getUsername() {
-        return currentConnection == null && fallbackToDatabase ? emailInput.getText() : usernameInput.getText();
+        boolean usingDatabase = currentConnection == null && fallbackToDatabase;
+        boolean usingOAuthEnterprise = currentConnection != null && !currentConnection.isActiveFlowEnabled();
+        if (usingDatabase || usingOAuthEnterprise) {
+            return emailInput.getText();
+        } else {
```

Comment:
are you just making code more understandable or is this required for https://github.com/auth0/Lock.Android/pull/410/files#diff-46c7332ff70ade6b4cc8caac4e443222R234 ?